bin/oe/rpc.rs

```diff
@@ -152,12 +195,24 @@ pub struct Dependencies<D: rpc_apis::Dependencies> {
     pub stats: Arc<RpcStats>,
 }
 
+fn format_ws_error(url: &str, e: Result<WsServer, rpc::ws::Error>) -> Result<WsServer, String> {
```

Comment:
IMO, name of the function does not reflect its meaning. I would suggest to change arguments in the following way making it exactly a formatter of ws errors
```suggestion
fn format_ws_error(url: &str, e: rpc::ws::Error) -> String 
```